turbo.json

```diff
@@ -26,62 +20,31 @@
       "cache": false
     },
     "build": {
-      "dependsOn": [
-        "generate-model",
-        "generate-protobuf",
-        "^build"
-      ],
-      "outputs": [
-        "dist"
-      ]
+      "dependsOn": ["generate-model", "generate-protobuf", "^build"],
+      "outputs": ["dist"]
     },
     "check": {
-      "dependsOn": [
-        "generate-model",
-        "generate-protobuf",
-        "^build"
-      ],
-      "outputs": [
-        "dist"
-      ]
+      "dependsOn": ["generate-model", "generate-protobuf", "^build"],
+      "outputs": ["dist"]
     },
     "generate-model": {
-      "dependsOn": [
-        "^generate-model"
-      ]
+      "dependsOn": ["^generate-model"],
+      "outputs": ["src/generated"]
     },
     "generate-protobuf": {
-      "dependsOn": [
-        "^generate-protobuf"
-      ],
-      "outputs": [
-        "src/gen"
-      ]
+      "dependsOn": ["^generate-protobuf"],
+      "outputs": ["src/gen"]
     },
     "test": {
-      "dependsOn": [
-        "build"
-      ]
+      "dependsOn": ["build"]
     },
     "lint": {
-      "dependsOn": [
-        "generate-model",
-        "generate-protobuf",
-        "^build"
-      ],
-      "outputs": [
-        "dist"
-      ]
+      "dependsOn": ["build"],
+      "outputs": ["dist"]
     },
     "lint:autofix": {
-      "dependsOn": [
-        "generate-model",
-        "generate-protobuf",
-        "^build"
-      ],
-      "outputs": [
-        "dist"
-      ]
+      "dependsOn": ["build"],
+      "outputs": ["dist"]
```

Comment:
Maybe we can speed up the linting making it depend on `check` instead of `build`?
Check should be faster than build and also be enough to make the linting work as expected (I think)

Reply:
I have some problems using check instead of build. Probably because we need to build package like `models`, `commons` etc and the check command is not enough